### DISCUSSION TITLE
Panel Field Overrides: Add support for regex value mapping on time fields

### DISCUSSION THREAD

**Discussion by axelavargas** [grafana/grafana#46145] [Dashboards, Panels & Visualizations]
Grafana does not support regex value mappings on time fields, 

![RegexValueTimeField](https://user-images.githubusercontent.com/239999/156531961-3f4e62cf-5e12-4245-9373-159d35c75c24.png)

Here is also the [dashboard with the current behavior](https://play.grafana.org/d/QMjKLHf7z/value-mapping-capture-groups?orgId=1)

**I am opening this discussion because we (User Essentials Grafana Team) are not sure how relevant this feature is to the community.**

**Comment by axelavargas** [comment]
There is a workaround found by @annelaurefroment 👏🏾 

> I don't think we need to add support for regex on time fields at all. There is a transformation called "Convert field" type that the user can use to convert the field from time to string, and then the value mapping works like a charm:
> ![image](https://user-images.githubusercontent.com/49911675/156585723-a04c99ca-c9d1-4b4d-9644-9487def29f43.png)

I can't close the discussion (only delete it), so I'll leave it open, for future reference.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
